### PR TITLE
Add description to created node images

### PR DESCRIPTION
**Pull request by fejta**
Make it a little easier to see who to contact about important node e2e images.

### CODE REVIEW

**Comment by dchen1107** [comment]
LGTM

Thanks for this improvement.

File: test/e2e_node/jenkins/copy-e2e-image.sh [hunk 1]
```diff
@@ -16,10 +16,15 @@
 
 # Usage: copy-e2e-image.sh <image-name> <from-project-name> <to-project-name>
 
+# See *.properties for list of images to copy,
+# typically from kubernetes-node-e2e-images
+
 set -e
 set -x
 
 echo "Copying image $1 from project $2 to project $3..."
```
Comment (lavalamp) [inline test/e2e_node/jenkins/copy-e2e-image.sh]: store the args into named variables so this script is readable with less context.

File: test/e2e_node/jenkins/copy-e2e-image.sh [hunk 2]
```diff
@@ -16,10 +16,15 @@
...
 gcloud compute --project $3 disks create $1 --image=https://www.googleapis.com/compute/v1/projects/$2/global/images/$1
-gcloud compute --project $3 images create $1 --source-disk=$1
-gcloud compute --project $3 disks delete $1
+gcloud compute --project $3 images create $1 \
+  --source-disk=$1 \
+  --description="Cloned from projects/$2/global/images/$1 by $USER on $(date)"
+gcloud -q compute --project $3 disks delete $1
```
Comment (lavalamp) [inline test/e2e_node/jenkins/copy-e2e-image.sh]: what's -q do?

Comment (fejta) [inline test/e2e_node/jenkins/copy-e2e-image.sh]: Skips asking for confirmation. We don't want the disk (created in line 26) but the easiest way to create an image is from a disk. So we use the temporary disk to create the image.

**Comment by fejta** [comment]
We're having problems tracking which resources are leaked vs used for testing. Adding 1.3 milestone per conversation with @lavalamp to this to make sure future test resources have contact info associated with them.

**Pull request merged**